### PR TITLE
wait for finalizers to release all garbage at the end of the build

### PR DESCRIPTION
**Pull request by asecchia**

### CODE REVIEW

File: src/Compilers/Server/ServerShared/ServerDispatcher.cs [hunk 1]
```diff
@@ -236,6 +236,8 @@ private void HandleCompletedGCTask()
         {
             _gcTask = null;
```
Comment (agocke) [inline src/Compilers/Server/ServerShared/ServerDispatcher.cs]: While you're here, we might as well do a LOH compaction. Would you mind adding that request?

Comment (jaredpar) [inline src/Compilers/Server/ServerShared/ServerDispatcher.cs]: What API are you looking to call for this?  Not familiar with how to invoke this.

Comment (jaredpar) [inline src/Compilers/Server/ServerShared/ServerDispatcher.cs]: @agocke

Comment (agocke) [inline src/Compilers/Server/ServerShared/ServerDispatcher.cs]: @jaredpar https://msdn.microsoft.com/en-us/library/system.runtime.gcsettings.largeobjectheapcompactionmode(v=vs.110).aspx

**Comment by jaredpar** [comment]
retest  windows_release_unit64_prtest please

**Comment by CyrusNajmabadi** [comment]
test windows_release_unit64_prtest please

**Comment by jaredpar** [comment]
Thanks!

**Pull request merged**